models/metrics/metric_unique_users_count.sql

```diff
@@ -0,0 +1,65 @@
+{{ 
+    config(
+        materialized='table',
+        tags=['metrics'],
+        cluster_by=['metric_date']
+    )
+}}
+
+with
+
+min_address as (
+
+    select 
+        from_address,
+        min(block_timestamp) as min_timestamp
+    from {{ ref("txs") }}
+    group by 1
+
+),
+
+daily as (
+
+    select
+        date_trunc('day', min_timestamp) as metric_date,
+        'daily' as metric_period,
+        count(from_address) as unique_users_count,
+        sum(count(from_address)) over (order by metric_date) as unique_users_cumulative
```

Comment:
you can directly use `unique_users_count` from above
```suggestion
        sum(unique_users_count) over (partition by metric_period order by metric_date) as unique_users_cumulative
```